scripts/rabbitmq-server-ha.ocf

```diff
@@ -260,6 +262,16 @@ ${OCF_RESKEY_binary} should listen on this port
 <content type="boolean" default="${OCF_RESKEY_node_port_default}" />
 </parameter>
 
+<parameter name="vhost" unique="0" required="0">
+<longdesc lang="en">
+Main virtual host used in the cluster, guaranteed to not suffer data loss. Other
+virtual hosts may exist, but some scenarios can lead to data loss for them. This
+must be correctly configured for the cluster to behave as intended.
+</longdesc>
+<shortdesc lang="en">Main virtual host</shortdesc>
```

Comment:
In case you are looking for naming recommendations, I'd use "default vhost" — whiles still not very precise — instead of "main vhost."